exchanges/persisted-fetch/src/persistedFetchExchange.ts

```diff
@@ -46,13 +47,14 @@ export const persistedFetchExchange = (
   const preferGetForPersistedQueries = !!options.preferGetForPersistedQueries;
   const enforcePersistedQueries = !!options.enforcePersistedQueries;
   const hashFn = options.generateHash || hash;
+  const enableForMutation = options.enableForMutation || false;
   let supportsPersistedQueries = true;
 
   return ops$ => {
     const sharedOps$ = share(ops$);
     const fetchResults$ = pipe(
       sharedOps$,
-      filter(operation => operation.kind === 'query'),
+      filter(operation => enableForMutation || operation.kind === 'query'),
```

Comment:
```suggestion
      filter(operation => (enableForMutation && operation.kind === 'mutation') || operation.kind === 'query'),
```

Reply:
I refactored this part of the code to use a single `operationFilter` method. This will prevent mismatch between the filter in and filter out logic of the exchange